src/main/java/core/basesyntax/model/Bet.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class Bet {
+    private int bet;
+    private double risk;
+    
+    public Bet(int bet, double risk) {
+        this.bet = bet;
+        this.risk = risk;
+    }
+    
+    public int getBet() {
+        return bet;
+    }
+    
+    public void setBet(int bet) {
+        this.bet = bet;
+    }
+    
+    public double getRisk() {
+        return risk;
+    }
+    
+    public void setRisk(double risk) {
+        this.risk = risk;
+    }
+    
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (!(o instanceof Bet)) {
```

Comment:
why instanceof?

Reply:
I applied standard implementation of equals and hashCode and it makes through instanceof

Reply:
changed it to class.equal(class)

Reply:
@Ketch92 let's use another implementation for autogeneration, there are a number of them available